client/performance-profiler/components/core-web-vitals-display/core-web-vitals-details.tsx

```diff
@@ -62,9 +64,9 @@ export const CoreWebVitalsDetails: React.FC< CoreWebVitalsDetailsProps > = ( {
 	let metricsData: number[] = history?.metrics[ activeTab ] ?? [];
 	let dates = history?.collection_period ?? [];
 
-	// last 8 weeks only
-	metricsData = metricsData.slice( -8 );
-	dates = dates.slice( -8 );
+	const weeksToShow = isMobile ? 6 : 8;
```

Comment:
What do you think about changing the [text here](https://github.com/Automattic/wp-calypso/pull/94487/files#diff-bd07663207e6e898ab2fe1cf62fe88d1f1cb1153cab6e42e39a4ddbbe541d609R159) depending on the points displayed? It could be confusing when moving from desktop to mobile, but it will be consistent with the number of points 🤔

Reply:
I updated the template in [52e76e9](https://github.com/Automattic/wp-calypso/pull/94487/commits/52e76e9ae6465d4a158ed601ca662cc4e0464c5d)